src/Select.js

```diff
@@ -59,10 +64,17 @@ class Select extends React.Component {
 			'onOptionRef',
 			'removeValue',
 			'selectValue',
+			'setAutofill',
```

Comment:
Looks like these four don't have to be here, since they're all called directly, and not used as event handlers. So no need for binding them.